src/pick.c

```diff
@@ -729,3 +792,15 @@ free_choices(void)
 	free(choices.v);
 	free(input.string);
 }
+
+int
+isu8cont(unsigned char c)
```

Comment:
Does this function return whether or not the byte is the continuation of an utf-8 character?